### ISSUE TITLE
Installation in editable/development mode does not work properly

### ISSUE THREAD

**Issue by garrelt** [sambit-giri/tools21cm#25]
The documentation suggests installing the package as `pip install -e /path/to/tools21cm [--user]` to allow doing development work on it. However, if you do this (or any other type of development install such as `python setup.py develop` or `conda develop /path/to/tools21cm`, you cannot run `import tools21cm` from within python. The reason is that the source directory is called `t2c` and not `tools21cm`. If you change the name of the source directory to `tools21cm` the import command works again. If you do not, you can also run `import t2c` which does work. However, I am not sure if we want to have a different import command for the development version.

If there are good reasons not to rename the directory we could perhaps add a soft link?

**Comment by sambit-giri** [comment]
`pip install -e /path/to/tools21cm` is supposed to create `tools21cm.egg-info` and then python should import this. This used to work, but I also have problems now. I am figuring it out now.

By running `import t2c`, it would import the folder `t2c`.  Renaming would be a quick fix but it would work only when one is inside the `tools21cm` folder.

**Comment by sambit-giri** [comment]
I have made changes in the structure of the package and now using `setuptools` instead of `distutils`. This seems to resolve the problem for me. Can you check with the updated version?

**Comment by sambit-giri** [comment]
I have changed a few other things. Can you try the latest version?

**Comment by garrelt** [comment]
I created a new conda environment, based on python 3.10.
I git cloned your distribution: `git clone git://github.com/sambit-giri/tools21cm.git`
I installed it with pip: `pip install -e /home/gmell/Science/Codes/tools21cm_new/tools21cm/`
I changed directory and ran python and imported tools21cm: `python -c 'import tools21cm'`
All this works without error messages. I then ran the tests which all past successfully. There are some warnings:
